swc/aeon/io/api.py

```diff
@@ -100,6 +132,9 @@ def load(root, reader, start=None, end=None, time=None, tolerance=None, epoch=No
     :param Reader reader: A data stream reader object used to read chunk data from the dataset.
     :param datetime, optional start: The left bound of the time range to extract.
     :param datetime, optional end: The right bound of the time range to extract.
+    :param str, optional inclusive: Specifies whether the ``start`` and ``end`` bounds are inclusive or
+        exclusive. This is only applicable when both ``start`` and ``end`` are provided. 
```

Comment:
```suggestion
        exclusive. This is only applicable when ``start`` and/or ``end`` are provided. 
```

Reading the code it looks like filter gets called when any one of them is defined. Not sure if we have tests covering all those edge cases, but assuming this is true I would suggest rewording as above?